src/add-layout/files/src/app/shared/components/header/header.component.scss

```diff
@@ -1,33 +1,33 @@
 @import "../../../../themes/generated/variables.base.scss";
+@import "../../../../dx-styles.scss";
 
 :host {
-  display: block;
   flex: 0 0 auto;
   z-index: 1;
-
   box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
 
-  ::ng-deep .dx-toolbar .dx-toolbar-item-content > .menu-button .dx-icon {
+  ::ng-deep .dx-toolbar .dx-toolbar-item.menu-button>.dx-toolbar-item-content .dx-icon {
     color: $base-accent;
   }
 }
 
-::ng-deep .dx-toolbar .dx-toolbar-items-container .dx-toolbar-after {
+::ng-deep .dx-toolbar.header-toolbar .dx-toolbar-items-container .dx-toolbar-after {
   padding: 0 40px;
 
   :host-context(.screen-x-small) & {
     padding: 0 20px;
   }
 }
 
-::ng-deep .menu-button {
-  margin: 0 12px;
+::ng-deep .dx-toolbar .dx-toolbar-item.menu-button {
```

Comment:
This selector can affect on all widgets within element

Reply:
Fixed